tests/transfer_test.go

```diff
@@ -472,6 +480,7 @@ var _ = Describe("[rfe_id:5630][crit:high]ObjectTransfer tests", func() {
 			rq.Spec.Hard[corev1.ResourceRequestsStorage] = *resource.NewQuantity(bq, resource.DecimalSI)
 			rq, err = f.K8sClient.CoreV1().ResourceQuotas(targetNs.Name).Update(context.TODO(), rq, metav1.UpdateOptions{})
 			Expect(err).ToNot(HaveOccurred())
+			Eventually(quotaUpdated, 2*time.Minute, 2*time.Second).Should(BeNumerically("==", 1))
```

Comment:
Maybe also check this before updating the quota? I would also add some comments explaining what we are doing here, the test seems a little bit confusing.

Reply:
Yep, changed the test name to better reflect the steps

Reply:
No need to check before updating, just after applying changes(create/update)